### DISCUSSION TITLE
How to create optional many-to-one records with no relation?

### DISCUSSION THREAD

**Discussion by susiyaki** [prisma/prisma#11041] [Q&A] [answered]
Hi!

I want to execute following query but error occurred.
I'm going to set relationship after upload image.

Query
```
prisma.itemImage
          .create({ data: { } })
```

Error
```
Argument item for data.item is missing.
```

Schema
```
model Item {
  id                   Int     @id @default(autoincrement())
  name                 String
  serialNumber         String?
  description          String?
  price                Int?

  thumbnail   ItemThumbnailImage @relation(fields: [thumbnailId], references: [id])
  thumbnailId Int                @unique

  // many
  images   ItemImage[]

  createdAt DateTime @default(now())
  updatedAt DateTime @default(now()) @updatedAt
}

model ItemImage {
  id Int @id @default(autoincrement())

  url      String?
  isPublic Boolean @default(false)

  // one
  item     Item?   @relation(fields: [itemId], references: [id])
  itemId   Int?

  createdAt DateTime @default(now())
  updatedAt DateTime @default(now()) @updatedAt
}

```

**Comment by susiyaki** [comment, marked as answer]
I fixed it.
It is just a cache problem.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
